### PR TITLE
Remove unecessary xstream dependency

### PR DESCRIPTION
**Pull request by rdenarie**

### CODE REVIEW

File: packaging/plf-dependencies/pom.xml [hunk 1]
```diff
@@ -182,6 +182,10 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
           <artifactId>xpp3</artifactId>
           <groupId>xpp3</groupId>
         </exclusion>
+        <exclusion>
```
Comment (boubaker) [inline packaging/plf-dependencies/pom.xml:185]: I think we should add this exclusion in the place where picocontainer dependency and version are declared

Comment (rdenarie) [reply packaging/plf-dependencies/pom.xml:185]: Do you mean here : https://github.com/Meeds-io/maven-depmgt-pom/blob/feature/microservices/pom.xml#L1327-L1340

Comment (boubaker) [reply packaging/plf-dependencies/pom.xml:185]: @rdenarie wasn't it sufficient to declare the exclusion here :
https://github.com/Meeds-io/maven-depmgt-pom/pull/17/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R1331 
?

Comment (boubaker) [reply packaging/plf-dependencies/pom.xml:185]: > Do you mean here : https://github.com/Meeds-io/maven-depmgt-pom/blob/feature/microservices/pom.xml#L1327-L1340

Yep

Comment (rdenarie) [reply packaging/plf-dependencies/pom.xml:185]: yes, I'm testing removing this PR. It seems ok

**Pull request closed without merging**